specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json

```diff
@@ -15,6 +15,47 @@
     "application/json"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerRegistry/registries/{registryName}/importImage": {
+      "post": {
+        "tags": [
+          "Registries"
+        ],
+        "description": "Copies an image to this registry from the specified registry.",
+        "operationId": "Registries_ImportImage",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/RegistryNameParameter"
+          },
+          {
+            "name": "importParameters",
```

Comment:
Can we name this `importImageParameters`? `importParameters` sounds too general.

Reply:
Looking at it I think it was too redundant (it was on an operation named importImage), so I've renamed it to parameters.